### ISSUE TITLE
The accuracy of the model trained from scratch is very low

### ISSUE THREAD

**Issue by jimmery-XUT** [tensorflow/models#9534]
Run train_image_classifer.py with default parameters .Run eval_image_classifer.py ,the accuracy rate is only 46%. How can I adjust the parameters similar to the learning rate to improve the accuracy. I just adjusted the learning rate, but the effect was not good.Do you have any good suggestions？

**Comment by ravikyram** [comment]
@jimmery-XUT 

Please, fill [issue template.](https://github.com/tensorflow/models/issues/new/choose).
Please, let us know which pretrained model you are using and share related code .Thanks!

**Comment by jimmery-XUT** [comment]
The  model trained from scratch is inception_resnet_v2   ,dataset is cifar10，max_number_of_steps is 10000.The loss has been fluctuating around 4.0 since step 2. After 10000 iterations, the loss has not decreased much. What's the matter? How to solve it?

INFO:tensorflow:Recording summary at step 0.
INFO:tensorflow:global step 2: loss = 4.8855 (0.211 sec/step)
INFO:tensorflow:global step 4: loss = 4.4874 (0.205 sec/step)
INFO:tensorflow:global step 6: loss = 4.7496 (0.205 sec/step)
INFO:tensorflow:global step 8: loss = 4.5145 (0.209 sec/step)
INFO:tensorflow:global step 10: loss = 4.5988 (0.204 sec/step)
INFO:tensorflow:global step 12: loss = 4.8288 (0.207 sec/step)
INFO:tensorflow:global step 14: loss = 4.3772 (0.204 sec/step)
INFO:tensorflow:global step 16: loss = 4.8843 (0.204 sec/step)
INFO:tensorflow:global step 18: loss = 4.6823 (0.210 sec/step)
INFO:tensorflow:global step 20: loss = 4.2232 (0.206 sec/step)
INFO:tensorflow:global step 22: loss = 5.0554 (0.216 sec/step)
INFO:tensorflow:global step 24: loss = 4.3586 (0.205 sec/step)
INFO:tensorflow:global step 26: loss = 4.6943 (0.204 sec/step)
INFO:tensorflow:global step 28: loss = 4.5133 (0.209 sec/step)
INFO:tensorflow:global step 30: loss = 4.1489 (0.207 sec/step)
INFO:tensorflow:global step 32: loss = 5.4161 (0.207 sec/step)
.......
INFO:tensorflow:global step 9970: loss = 3.3127 (0.207 sec/step)
INFO:tensorflow:global step 9972: loss = 3.0967 (0.208 sec/step)
INFO:tensorflow:global step 9974: loss = 3.1755 (0.211 sec/step)
INFO:tensorflow:global step 9976: loss = 3.5761 (0.209 sec/step)
INFO:tensorflow:global step 9978: loss = 3.8318 (0.210 sec/step)
INFO:tensorflow:global step 9980: loss = 2.9371 (0.210 sec/step)
INFO:tensorflow:global step 9982: loss = 4.0937 (0.209 sec/step)
INFO:tensorflow:global step 9984: loss = 4.0091 (0.211 sec/step)
INFO:tensorflow:global step 9986: loss = 3.4477 (0.207 sec/step)
INFO:tensorflow:global step 9988: loss = 3.4877 (0.213 sec/step)
INFO:tensorflow:global step 9990: loss = 4.0967 (0.209 sec/step)
INFO:tensorflow:global step 9992: loss = 4.6254 (0.211 sec/step)
INFO:tensorflow:global step 9994: loss = 3.9375 (0.212 sec/step)
INFO:tensorflow:global step 9996: loss = 2.9722 (0.210 sec/step)
INFO:tensorflow:global step 9998: loss = 3.4542 (0.211 sec/step)
INFO:tensorflow:global step 10000: loss = 4.9306 (0.208 sec/step)
INFO:tensorflow:Stopping Training.
INFO:tensorflow:Finished training! Saving model to disk.
